transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,26 @@
+{{
+config({
+ "tags":"hourly",
+})
+}}
+
+WITH MM_TELEMETRY_PROD AS 
+  {{ dbt_utils.union_relations(
+    relations=[
+      source('mm_telemetry_prod', 'event'),
+]
+, include=["context_page_referrer", "channel", "context_app_namespace", "user_actual_id", 
+    "context_library_name", "type", "context_app_version" , "user_actual_role", "context_page_url"
+, "context_os_name" , "context_page_title"  ,  "context_app_build", "context_page_search" 
+, "context_library_version",  "context_useragent", "context_app_name", "context_locale"
+, "context_screen_density" , "context_page_path", "context_os_version"  , "category"  , "duration"  
+, "num_of_request", "max_api_resource_size", "longest_api_resource_duration", "user_id", "userid" 
+, "root_id", "post_id", "sort", "team_id", "version", "keyword", "count"
+, "gfyid", "field", "plugin_id" , "installed_version", "group_constrained", "value", "include_deleted" 
+, "role", "privacy", "scheme_id" , "warnmetricid"  , "metric" , "error"
+, "num_invitations_sent", "num_invitations" , "channel_sidebar" , "app" , "method" , "remaining" 
+, "screen" , "filter" , "version" , "uuid_ts", "id", "anonymous_id", "received_at", "sent_at", "original_timestamp", "timestamp", 
+        "context_ip", "event", "event_text"], where = " CATEGORY = 'performance'",
+  )}}
+
+SELECT * FROM MM_TELEMETRY_PROD
```

Comment:
Using `dbt_utils.union_relations` here for a single table will do the work. However the name can be a bit confusing for anyone who reads the code. One thing that can improve the readability of the code is to change this to something like this (untested):
```suggestion
{%- set include_columns = ["context_page_referrer", "channel", "context_app_namespace", "user_actual_id", 
    "context_library_name", "type", "context_app_version" , "user_actual_role", "context_page_url"
, "context_os_name" , "context_page_title"  ,  "context_app_build", "context_page_search" 
, "context_library_version",  "context_useragent", "context_app_name", "context_locale"
, "context_screen_density" , "context_page_path", "context_os_version"  , "category"  , "duration"  
, "num_of_request", "max_api_resource_size", "longest_api_resource_duration", "user_id", "userid" 
, "root_id", "post_id", "sort", "team_id", "version", "keyword", "count"
, "gfyid", "field", "plugin_id" , "installed_version", "group_constrained", "value", "include_deleted" 
, "role", "privacy", "scheme_id" , "warnmetricid"  , "metric" , "error"
, "num_invitations_sent", "num_invitations" , "channel_sidebar" , "app" , "method" , "remaining" 
, "screen" , "filter" , "version" , "uuid_ts", "id", "anonymous_id", "received_at", "sent_at", "original_timestamp", "timestamp", "context_ip", "event", "event_text"] %-}
        
WITH performance_event AS (
    SELECT
      {{ get_rudderstack_columns() }},
        {% for column in include_columns %}
        {{ column }} AS {{ column }}
        {% if not loop.last %},{% endif %}
    {% endfor %}
    FROM
      {{{ source('mm_telemetry_prod', 'event') }}
    WHERE CATEGORY = 'performance'",
)

SELECT * FROM performance_events
```

This  code might seem a bit more, but it communicates clearly the intent of filtering some of the columns  from `event` table in order to keep only records where `category = 'performance'`.

There are also a few more improvements that can be done here. 
- `include_columns` contains a couple of columns that are already defined in `get_rudderstack_columns`, so they need to be removed from `include_columns`.
- `include_columns` contains a few columns that are reused across events in `events` table. For example `category`, `user_id`, `user_actual_id`, `user_actual_role`, `category`, `type` should be all present. These columns can be extracted to a separate list or a macro similar to `get_rudderstack_columns`. This way they will be reusable whenever a new staging model that extracts specific events from `events` is added.